source/query/filter.txt

```diff
@@ -47,6 +47,283 @@ Set Query Filter
 
    .. include:: /includes/extracts/query-bar-results.rst
 
+Examples
```

Comment:
[praise]

I think this much more scannable than the previous version and the right-hand nav titles are a clear win. 👍